00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/Content Comments.md

```diff
@@ -22,6 +24,8 @@ Suggested standard:
 
 ## Agree a standard begin/end prefix for marking 'do not edit' sections
 
+**Note:** *See [Agree a standard begin/end prefix for marking 'do not edit' sections · Issue #351](https://github.com/obsidian-community/obsidian-hub/issues/351) for discussion and implementation progress on this.*
+
 We use these markers in Author pages, and they do not seem to be well-understood:
```

Comment:
I stumbled over this part earlier. The "do not seem to be well-understood" seems to out of place without an explanation of *why* they are misunderstood.

Could we maybe prepend a short explanation/description here or at the top of this note on why we use these markers in general? Something like:
> We use these markers to highlight sections which should not be edited manually but are instead created automatically by our [[GitHub Actions for the Hub]].

Reply:
That's a great point. I've pushed a change based on your text, and credited you as co-author. Thanks!